pkgs/development/python-modules/bayesian-optimization/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, scikitlearn
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "bayesian-optimization";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0j0cwicq6y4bbhpgwppay4cnv347zdmw2lk9k4gl7zn3pl6kkfmp";
+  };
```

Comment:
doesn't include tests, please check out from github:
```
============================ test session starts ==============================
platform linux -- Python 3.7.6, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /build/bayesian-optimization-1.0.1
collected 0 items

============================ no tests ran in 0.01s =============================
```

important for ensuring the package is still validated when bumping dependencies. scikitlearn and scipy are notorious for breaking APIs

Reply:
Fetching from github solves the problem. Thanks.